unreleased.rst

```diff
@@ -27,3 +27,7 @@ HEAD — ongoing
   default, but will revert to the extension bundled with the DAML SDK if installation
   fails. You can get the old default behavior of always using the bundled extension
   by running ``daml studio --replace=newer`` or ``daml studio --replace=always`` instead.
+- [DAML Studio] The gRPC message size limit can now be configured in
```

Comment:
just tiny suggestion, using active voice instead of passive to be more direct
```suggestion
- [DAML Studio] You can now configure the gRPC message size limit in
```